be/src/exec/pipeline/exchange/local_exchange_source_operator.cpp

```diff
@@ -43,6 +49,28 @@ bool LocalExchangeSourceOperator::has_output() const {
            (_is_finished && _partition_rows_num > 0);
 }
 
+void LocalExchangeSourceOperator::finish_backward(RuntimeState* state) {
```

Comment:
I think we no need to add a `finish_backward` method, you could implement this logic in `finish`.

Reply:
finish func is invoked by the current operator to its following operator that the EOS is comming, but the following operator should still process the buffered chunks inside.
finish_backward is invoked by the current operator to preceding operators that they should be finished, the buffered data is needless.